optimum/exporters/openvino/model_patcher.py

```diff
@@ -1181,11 +1180,7 @@ def _phi3_self_attn_sdpa_forward(
             use_cache=use_cache,
         )
 
-    # TO DO: remove llama imports when transformers with phi3 support will be released
```

Comment:
also I think we could remove `_phi3_self_attn_sdpa_forward` completly https://github.com/huggingface/transformers/blob/ccdabc5642bf84849af93f591e207dc625c8e1e1/src/transformers/models/phi3/modeling_phi3.py#L729
wdyt @eaidova ?

Reply:
your changes will not allow to run this model with trust_remote_code = True, because the goal of these changes is resolve buggy implementation https://huggingface.co/microsoft/Phi-3-mini-4k-instruct/blob/main/modeling_phi3.py

Reply:
the problem that phi3 has SDPA disabled in transformers https://github.com/huggingface/transformers/blob/ccdabc5642bf84849af93f591e207dc625c8e1e1/src/transformers/models/phi3/modeling_phi3.py#L923, so this code is never reached on our side even if we explicitly set attention implementation sdpa via lfrom_pretrained arguments

Reply:
ok that's clear let's keep it then, added https://github.com/huggingface/optimum-intel/pull/784/commits/280543b757a38ef7076d08f5ccac5677d4fbb9dc to enable `trust_remote_code=True` (in which case the transformers version is not guaranteed)